tests/test_urdf.py

```diff
@@ -0,0 +1,39 @@
+
+import math
+import os
+
+import pytest
+import xacro
+from urdf_parser_py.urdf import URDF
+
+model_data = [
+    ("med7", 25.5),
+    ("med14", 32.3),
+    ("iiwa7", 23.9),
+    ("iiwa14", 29.9)
+]
+
+@pytest.mark.parametrize("model, mass", model_data)
+def test_urdf_mass(model: str, mass: float, abs_tol: float=1.e-5) -> None:
+    r"""Tests the total mass withtin the URDF files against
```

Comment:
Consider rewording: "Compares the the total mass within ..."

Reply:
updated doc string